traffic_router/core/src/main/java/com/comcast/cdn/traffic_control/traffic_router/core/edge/Location.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2015 Comcast Cable Communications Management, LLC
```

Comment:
I'm not a lawyer, but none of the other files in here have this Comcast copyright.

Reply:
Good catch. I just realized that 'comcast' is part of the package name for ALL of the Java classes. I guess we'll have to resolve that some day.